wpk/README.md

```diff
@@ -81,10 +74,14 @@ To build  a WPK package, it is necessary to generate a X509 certificate and CA,
 - [OVA](/ova/README.md)
 - [KibanaApp](/wazuhapp/README.md)
 - [SplunkApp](/splunkapp/README.md)
+<<<<<<< HEAD
```

Comment:
Wrong conflict solution

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/188/commits/e31f2c76f7e659237bf2ed8a911d9d82bf4dd5db